### PR TITLE
Add Keyma.sh

### PR DESCRIPTION
**Pull request by CameronCT**

### CODE REVIEW

File: websites/K/Keymash/dist/metadata.json [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+  "$schema": "https://schemas.premid.app/metadata/1.0",
+  "author": {
+    "name": "GNiK",
+    "id": "83888634339528704"
+  },
+  "url": "keyma.sh",
+  "description": {
+    "en": "Take your typing to the next level! An online multiplayer typing game where you can go head to head versus players around the world."
+  },
+  "service": "KEYMA.SH",
```
Comment (Bas950) [inline websites/K/Keymash/dist/metadata.json:11]: Service name and folder name must be the same

File: websites/K/Keymash/dist/metadata.json [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+  "version": "0.0.1",
+  "logo": "https://i.imgur.com/vDGvohb.png",
```
Comment (Bas950) [inline websites/K/Keymash/dist/metadata.json:13]: Version 1.0.0

**Review by Bas950 — CHANGES_REQUESTED** [review]
Upload proof of it working

**Comment by ririxidev** [comment]
no response.

**Pull request closed without merging**